solutions/src/main/java/ru/sberbank/school/task02/enums/Symbols.java

```diff
@@ -0,0 +1,22 @@
+package ru.sberbank.school.task02.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import ru.sberbank.school.task02.util.Symbol;
+
+/**
+ * Класс для работы с валютными парами
+ * Created by Gregory Melnikov at 03.05.2019
+ */
+
+@Getter
+@AllArgsConstructor
+public enum Symbols {
+
+    USD_RUB ("USD/RUB", Symbol.USD_RUB, Symbol.USD_RUB.isCross()),
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a268990300>*